tutorials/introductory/getting_started.py

```diff
@@ -0,0 +1,842 @@
+
+# The unpacked wedges variable serve as handles for legend.
+# Info from categories correspond to respective wedge instead of redundant
+# labeling from previous pie chart.
+# Legend has title keyword argument.
+# Keyword argument bbox_to_anchor with loc places legend at specific point.
+# Tuple floats are coordinates for Figure as row and column of Axes.
+# Keyword argument loc works with bbox_to_anchor to determine part of legend
+# for placement. Without bbox_to_anchor, Matplotlib automatically manages
+# coordinates in relation to parameters of loc.
+ax.legend(wedges,
+          categories,
+          title='Categories',
+          bbox_to_anchor=(0.125, 0.5),
+          loc='center right')
+
+ax.set_title('Average Monthly Income Expenses')
+ax.axis('equal')
+
+# The Figure method tight_layout() manages space between all Artists to
+# maximize visiblity on Figure. This method also contains various
+# parameters for configuration.
+fig.tight_layout()
+
+plt.show()
+
+##############################################################################
+#
+# .. note::
+#
+#   The ``partial`` function in functools works as a callable for simplifying
+#   a function's arguments. In the ``autopct`` keyword argument, only one
+#   argument is provided, the data acting as a percentage equivalent. The
+#   ``autopct_format`` function requires two arguments, so ``partial`` takes
+#   the argument for ``group`` and sets it to ``budget``. This smaller
+#   signature object then behaves as the same function with one fewer argument.
+#   For details about the functools module, see
+#   `functools
+#   <https://docs.python.org/3/library/functools.html#module-functools>`_.
+#
+# Multiple Graphs within a Figure
+# -------------------------------
+#
+# For multiple graphs using a single Figure, explicit and implicit approaches
+# use a similar convention for mapping out multiple Axes. Matplotlib manages
+# more than one Axes in a two-dimensional matrix. They are arranged by row
+# amount and then by column amount.
+#
+# Implicit coding uses a separate method with a similar name. The method
+# ``plt.subplot`` also includes a third argument to represent the specific
+# Axes involved.
+#
+# When looking for more complex solutions to multiple graphs within a Figure,
+# use the :class:`matplotlib.gridspec.GridSpec` module to organize the layout.
+#
+# Explicit
+# ^^^^^^^^
+
+# Explicit with OOP
+
+# Figure and two Axes unpacked from matrix as row (1) & column (2).
+# Keyword arguments provide additional details of sharing Y-Axis, Figure size
+# and layout formatting.
+fig, (ax1, ax2) = plt.subplots(1, 2,
+                               sharey='row',
+                               figsize=[8, 4],
+                               constrained_layout=True)
+
+# Explicit Figure object has separate method for title.
+fig.suptitle('Personal Financial Tracking \'09 - \'10')
+
+# First explicit Axes object plots data with additional keyword arguments.
```

Comment:
I'm not sure how calling out that it's the first or second, or that it's explicit, helps here. These calls exist on all `Axes` objects (so maybe it makes sense to call that out on the second one, but I'm not entirely convinced.)